src/lib.rs

```diff
@@ -857,6 +808,7 @@ impl Decoder<Instruction> for InstDecoder {
             }
         }
 
+        instr.length = words.offset() as u8;
```

Comment:
this ........... is probably a yaxpeax-arch bug sigh sigh sigh

`offset` should certainly be a number of `Word`, not the number of bytes this particular `words` read to get a `Word`. oh well this computes the right length for now

Reply:
i made `Word` just u8 for the time being while i publish a fixed yaxpeax-arch and yank the right crates, so this'll work regardless of changes for `U{16,32,64}{le,be}`

Reply:
i did in fact fix yaxpeax-arch (see 0.2.3+), but i'm leaving this as `u8` words because it's less of a change that way.